editor/src/templates/editor-canvas.tsx

```diff
@@ -434,13 +434,15 @@ export function runLocalCanvasAction(
             action.interactionSession.interactionData.originalDragStart,
             action.interactionSession.interactionData.drag ?? zeroCanvasPoint,
           )
+          // FIXME This probably means we need both
```

Comment:
yeah, it totally means that! I think this should be fixed before merge, because this can lead to subtle but incredibly annoying bugs. Unfortunately I still didn't write the test cases yet, but I'll make a note to explicitly write a test that would break because of this change

Reply:
Ok this has now been fixed